jakarta.batch.official.tck/src/main/asciidoc/batch-tck-reference-guide.adoc

```diff
@@ -76,12 +76,7 @@ For the current TCK version, this can be done with either Java SE Version 11 or
 
 === Other Dependencies
 3. **Arquillian** - Since the EE Platform TCK uses Arquillian to execute tests within an Arquillian "container" for certifying against the EE Platform, you must configure an Arquillian https://arquillian.org/guides/developing_a_container_adapter/[adapter] for your target runtime.
-4. **Signature Test Tool** - No action is needed here, but we note that the signature files were built and should be validated with the Maven plugin with group:artifact:version coordinates: **org.netbeans.tools:sigtest-maven-plugin:1.6**, as used by the sample sigtest runner included in the TCK zip. This is a more specific direction than in earlier releases, in which it was left more open for the user to use a compatible tool.  Since there are small differences in the various signature test tools, we standardize on this version.
-
-
-
-
-
+4. **Signature Test Tool** - No action is needed here, but we note that the Java 11 and 17 signature files were built with the Maven plugin with group:artifact:version coordinates: **org.netbeans.tools:sigtest-maven-plugin:1.6**.  The Java 21 signature file was built with the newer plugin maintained by the EE4J organization, with coordinates:  **jakarta.tck:sigtest-maven-plugin:2.2**.  The sample sigtest runner included in the TCK zip also uses the EE4J plugin, and it acan be used to validate an implementation using one of the Java 11 or 17 signature files (built by the other plugin) as well as the newest Java 21 signature file.  Though there are other signature test tools available, one of these two tools should be used when executing this TCK.
```

Comment:
`*jakarta.tck:sigtest-maven-plugin:2.2*` should be good to use but we can always upgrade to newer versions if needed.  

I think that `*jakarta.tck:sigtest-maven-plugin:2.3*` will likely include the https://github.com/eclipse-ee4j/jakartaee-tck-tools/pull/30 change which changes the `-IgnoreJDKClass` option to no longer take the list of JDK classes to be specified, instead we will ignore the `java.*` + `javax.transaction.xa` which is more future proof (as long as the `*jakarta.tck:sigtest-maven-plugin:2.3*` code works on whichever future Java SE version).

`*jakarta.tck:sigtest-maven-plugin:2.3*` will also include https://github.com/eclipse-ee4j/jakartaee-tck-tools/pull/31 which addresses a java.util.ConcurrentModificationException failure in the PackageGroup.checkName() method (I am see this failure intermittently with Platform TCK testing).

Reply:
With the -IgnoreJDKClass option specified, we could then just use the `sigtest-1.6-batch.standalone.tck.sig-2.1-se11-OpenJDK-J9` file on Java 11/17/21

Reply:
That sounds useful... and appreciate the info.  It sounds like we can queue this up for a future enhancement, so I opened https://github.com/jakartaee/batch-tck/issues/73 and referenced this comment.

Since I have a typo in there anyway... `acan` ...let me come back and soften the language a bit.